src/components/ResourceList.tsx

```diff
@@ -490,7 +530,17 @@ const ResourceList: React.FC<ResourceListProps> = ({
       )}
       <div className="kuadrant-policy-list-body">
         <ListPageBody>
-          <ListPageFilter data={data} loaded={allLoaded} onFilterChange={onFilterChange} />
+          <div className="kuadrant-filter-bar" style={{ marginBottom: '16px', marginTop: '15px' }}>
+            <SearchInput
+              placeholder={t('Search across name, namespace and/or resource type')}
+              value={String(filters) || ''}
+              onChange={(_, value: string) => handleFilterChange(value)}
+              onClear={handleClear}
+              aria-label="Global search input"
+              style={{ marginRight: '8px', width: '350px' }}
```

Comment:
nit: move to class

Reply:
worky pretty good. Wonder if we should look at a toolbar instead in keeping with other resource views?

https://www.patternfly.org/components/toolbar/html-demos/#toolbar-attribute-value-search-filter-on-mobile
![Screenshot 2024-11-19 at 10 49 36](https://github.com/user-attachments/assets/2835a26c-9859-4343-b26e-f32b79c98475)